### DISCUSSION TITLE
Altering the bounds of the yaw optimisation

### DISCUSSION THREAD

**Discussion by henrysanders3** [NatLabRockies/floris#582] [Q&A]
Hi all,

After changing the minimum and maximum and maximum yaw angles in the yaw optimisation in the FLORIS tools files (e.g. 'yaw_optimizer_sr') the yaw optimisation code is still optimising between angles of 0-20 degrees. how can I alter this bounds for the optimisation of the yaw?

Many thanks

**Comment by Bartdoekemeijer** [comment]
Hi @henrysanders3. Can you provide an example code demonstrating the problem?

**Reply by henrysanders3** [reply]
![Screenshot 2023-02-16 133826](https://user-images.githubusercontent.com/78905038/219379913-efcd8ce6-e073-48fa-84b7-195bf5def79a.png)
 Above is the screenshot of the yaw_optimizer_sr file. i have changed the bound to -30 and 30 for the yaw angles however when the optimisation code is run, it still optimises between 0 and 25 degrees.

Below is the code being run to optimise the the yaw angles for the 25 turbine wind farm.

![image](https://user-images.githubusercontent.com/78905038/219380663-c91412c9-7cf7-4972-a4a3-965acf9ba3f7.png)

Thanks for your help!
